test/uk/gov/hmrc/customs/emailfrontend/connectors/EmailVerificationConnectorSpec.scala

```diff
@@ -70,50 +70,49 @@ class EmailVerificationConnectorSpec extends SpecBase with BeforeAndAfter {
 
     "the email is not verified" should {
       "return an EmailNotVerified response" in {
-        when(
-          mockHttpClient.POST[JsObject, EmailVerificationStateResponse](any,
-                                                                        any,
-                                                                        any)(
-            any,
-            any,
-            any[HeaderCarrier],
-            any
-          )
+
+        when(mockHttpClient.POST[JsObject, EmailVerificationStateResponse](any,
+          any,
+          any)(
+          any,
+          any,
+          any[HeaderCarrier],
+          any)
         ).thenReturn(Future.successful(Right(EmailNotVerified)))
 
-        val result =
-          connector.getEmailVerificationState("email-address").futureValue
+        val result = connector.getEmailVerificationState("email-address").futureValue
 
         result shouldBe Right(EmailNotVerified)
       }
     }
 
     "the email service provides an unexpected state" should {
       "return an EmailVerificationStateErrorResponse" in {
-        when(
-          mockHttpClient.POST[JsObject, EmailVerificationStateResponse](any,
-                                                                        any,
-                                                                        any)(
-            any,
-            any,
-            any[HeaderCarrier],
-            any
-          )
+
+        val errorCode = 500
+
+        when(mockHttpClient.POST[JsObject, EmailVerificationStateResponse](any,
+          any,
+          any)(
+          any,
+          any,
+          any[HeaderCarrier],
+          any)
         ).thenReturn(Future.successful(Left(
-          EmailVerificationStateErrorResponse(500, "Internal Server Error"))))
+          EmailVerificationStateErrorResponse(errorCode, "Internal Server Error"))))
```

Comment:
Please replace 500 with INTERNAL_SERVER_ERROR

Once done, remove val errorCode.  Same for line 107